### PR TITLE
Initial UI test to make sure login works

### PR DESCRIPTION
**Pull request by timja**
relates to https://github.com/jenkins-infra/account-app/issues/166

### CODE REVIEW

File: src/it/resources/org/jenkinsci/account/ui/test-data.ldif [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+o: Jenkins users
+dc: Jenkins-Ci
+description: Jenkins users
+
+#dn: cn=admin,dc=jenkins-ci,dc=org
+#objectClass: simpleSecurityObject
+#objectClass: organizationalRole
+#cn: admin
+#description: LDAP administrator
+#userPassword: {SSHA}yI6cZwQadOA1e+/f+T+H3eCQQhRzYWx0
```
Comment (lemeurherve) [inline src/it/resources/org/jenkinsci/account/ui/test-data.ldif:20]: Shouldn't this commented block be removed?

File: src/it/resources/org/jenkinsci/account/ui/test-data.ldif [hunk 2]
```diff
@@ -0,0 +1,59 @@
+#
```
Comment (lemeurherve) [inline src/it/resources/org/jenkinsci/account/ui/test-data.ldif:1]:
```suggestion
```
(nit)

**Review by lemeurherve — APPROVED** [review]
LGTM

File: src/it/resources/org/jenkinsci/account/ui/test-data.ldif (hunk 1 above)
Comment (timja) [reply src/it/resources/org/jenkinsci/account/ui/test-data.ldif:20]: I have a follow up PR https://github.com/timja/account-app/pull/1 which will adjust this so don't worry about it

**Review by dduportal — APPROVED** [review]

**Pull request merged**